### DISCUSSION TITLE
Converting existing PDF to QuestPDF

### DISCUSSION THREAD

**Discussion by stephhnavarro** [QuestPDF/QuestPDF#740] [Q&A]
I am retrieving documents from a blob storage container in Azure. I want to take the document and place its contents into an IDocument to then merge the document with other QuestPdf documents. Is this possible, or will I need the support from another library to accomplish this?

**Comment by girlpunk** [comment]
Hi,

Unfortunately merging or otherwise editing existing PDFs is not currently possible with this library (https://github.com/QuestPDF/QuestPDF/issues/113). You may be able to achieve what you're looking for with [PdfPig](https://uglytoad.github.io/PdfPig/)?
